Simplified/NYPLSettingsAccountDetailViewController.m

```diff
@@ -609,6 +605,22 @@ - (void)showLoginAlertWithError:(NSError *)error
   [self removeActivityTitle];
 }
 
+- (void)showLogoutAlertWithError:(NSError *)error
+{
+  NSString *title; NSString *message;
+  if (error.code == 401) {
+    title = @"Unexpected Credentials";
+    message = @"Your username or password may have changed since the last time you logged in.\n\nIf you believe this is an error, please contact your library.";
```

Comment:
Should we be using localized strings here?

Reply:
`NYPLAlertController alertWithTitle:` wraps the parameters with NSLocalizedString